python/hsfs/core/job.py

```diff
@@ -58,3 +62,35 @@ def executions(self):
     @property
     def href(self):
         return self._href
+
+    def run(self, await_termination=True):
+        """Run the job.
+
+        Runs the job, by default awaiting its completion.
+
+        !!! example
+            ```python
+            # connect to the Feature Store
+            fs = ...
+
+            # get the Feature Group instances
+            fg = fs.get_or_create_feature_group(...)
+
+            # insert in to feature group
+            job, _ = fg.insert(df, write_options={"start_offline_backfill": False})
+
+            # run job
+            job.run()
+            ```
+
+        # Arguments
+            await_termination: bool. Identifies if the client should wait for the job to complete, defaults to True.
```

Comment:
if you add the type annotation, you don't need to write it in text in the docstring
```suggestion
    def run(self, await_termination: bool = True):
        """Run the job.

        Runs the job, by default awaiting its completion.

        !!! example
            ```python
            # connect to the Feature Store
            fs = ...

            # get the Feature Group instances
            fg = fs.get_or_create_feature_group(...)

            # insert in to feature group
            job, _ = fg.insert(df, write_options={"start_offline_backfill": False})

            # run job
            job.run()
            ```

        # Arguments
            await_termination: Identifies if the client should wait for the job to complete, defaults to True.
```